src/main/java/org/example/CalendarManagement/calendarpersistence/model/Employee.java

```diff
@@ -87,23 +85,7 @@ public boolean isDeleted() {
         return isDeleted;
     }
 
-   /* public LocalDateTime getCreatedDateTime() {
-        return createdDateTime;
+    public void setDeleted(boolean deleted) {
```

Comment:
don't use setter, create a new employee out of existing object and save that, low priority